cypress/integration/seededFlows/adminFlows/users/userIndexView.spec.js

```diff
@@ -233,6 +270,29 @@ describe('User index view', () => {
       });
     });
 
+    describe('Empty state', () => {
```

Comment:
My read is that this duplicates line 174, but not quite.  How is this different from the above?

Reply:
@jeremyf it's different from the above in that it searches and filters for results that don't exist and returns an "empty state" view rather than results, for large screens rather than for small screens. There is a [slight difference in the searching and filtering](https://github.com/forem/forem/pull/17777/files#diff-dc32870eda99bf9b77089993f832ab6dcb8473a657bb2b8ddfc4e216b912a963R175) for large vs. small screens, which can be seen on line `174`. 

Separating out these tests for the different screen sizes is a pattern we follow throughout the Member Index View tests since we have built the Member Index View responsively. This pattern can be seen in this test, `userIndexView.spec.js`, outside of my additions.